Integer overflow when compiling for arm 32 arch

Getting the following error compiling for arm32 raspberry pi:
`/go/pkg/mod/github.com/wagslane/go-rabbitmq@v0.12.1/internal/dispatcher/dispatcher.go:48:18: cannot use maxRand - minRand (untyped int constant 9223372036854775807) as int value in argument to rand.Intn (overflows)`

consider changing from MaxInt64 to MaxInt

--- a/internal/dispatcher/dispatcher.go
+++ b/internal/dispatcher/dispatcher.go
@@ -43,7 +43,7 @@ func (d *Dispatcher) Dispatch(err error) error {
 
 // AddSubscriber -
 func (d *Dispatcher) AddSubscriber() (<-chan error, chan<- struct{}) {
-	const maxRand = math.MaxInt64
+	const maxRand = math.MaxInt
 	const minRand = 0
 	id := rand.Intn(maxRand-minRand) + minRand
 

